repos/rust-lang/calendar-generation.toml

```diff
@@ -0,0 +1,15 @@
+org = "rust-lang"
+name = "calendar-generation"
+description = "Generate iCalendar documents from calendar specifications in TOML. See rust-lang/calendar."
+bots = ["rustbot"]
+
+[access.teams]
+compiler = "write"
+infra = "write"
+
+[[branch-protections]]
+pattern = "main"
+ci-checks = ['continuous integration']
+dismiss-stale-review = false
+pr-required = true
```

Comment:
```
pr-required = true
review-required = true
```
are not actually valid TOML entries for `toml`, it's just an artifact of my extraction script.

`dismiss-stale-review = false` is the default option, so it doesn't have to be used here.

Reply:
Thanks, fixed.